templates/default/capacity-scheduler.xml.erb

```diff
@@ -33,7 +33,7 @@
 
   <property>
     <name>yarn.scheduler.capacity.resource-calculator</name>
-    <value>org.apache.hadoop.yarn.util.resource.DefaultResourceCalculator</value>
+    <value>org.apache.hadoop.yarn.util.resource.DominantResourceCalculator</value>
```

Comment:
The DominantResourceCalculator is not available in hadoop 2.4, http://hadoop.apache.org/docs/r2.4.1/hadoop-yarn/hadoop-yarn-site/CapacityScheduler.html, so you can override this in your yml files. Can you make this a parameter, and keep the defaultResourceCalculator?